<ui version="4.0" >
 <class>mainOpt</class>
 <widget class="QWidget" name="mainOpt" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>100</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>0</hsizetype>
    <vsizetype>0</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>310</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>310</width>
    <height>100</height>
   </size>
  </property>
  <property name="contextMenuPolicy" >
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>5</x>
     <y>83</y>
     <width>97</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>0</hsizetype>
     <vsizetype>0</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>Number of functions:</string>
   </property>
  </widget>
  <widget class="QLabel" name="functionsCount" >
   <property name="geometry" >
    <rect>
     <x>135</x>
     <y>81</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>0</string>
   </property>
   <property name="scaledContents" >
    <bool>true</bool>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter</set>
   </property>
   <property name="textInteractionFlags" >
    <enum>Qt::NoTextInteraction</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelIterationsValue" >
   <property name="geometry" >
    <rect>
     <x>104</x>
     <y>0</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>0</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buddy" >
    <cstring>iterationSlider</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="labelIterations" >
   <property name="geometry" >
    <rect>
     <x>4</x>
     <y>1</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>0</hsizetype>
     <vsizetype>0</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>Number of iterations:</string>
   </property>
   <property name="buddy" >
    <cstring>iterationSlider</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="labelWidth" >
   <property name="geometry" >
    <rect>
     <x>5</x>
     <y>58</y>
     <width>46</width>
     <height>14</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>0</hsizetype>
     <vsizetype>0</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>Width:</string>
   </property>
   <property name="buddy" >
    <cstring>spinBoxWidth</cstring>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxHeight" >
   <property name="geometry" >
    <rect>
     <x>230</x>
     <y>56</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="autoFillBackground" >
    <bool>false</bool>
   </property>
   <property name="buttonSymbols" >
    <enum>QAbstractSpinBox::PlusMinus</enum>
   </property>
   <property name="accelerated" >
    <bool>true</bool>
   </property>
   <property name="correctionMode" >
    <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
   </property>
   <property name="maximum" >
    <number>4000</number>
   </property>
   <property name="minimum" >
    <number>600</number>
   </property>
   <property name="value" >
    <number>600</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxWidth" >
   <property name="geometry" >
    <rect>
     <x>80</x>
     <y>56</y>
     <width>60</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>0</hsizetype>
     <vsizetype>0</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground" >
    <bool>false</bool>
   </property>
   <property name="buttonSymbols" >
    <enum>QAbstractSpinBox::PlusMinus</enum>
   </property>
   <property name="accelerated" >
    <bool>true</bool>
   </property>
   <property name="correctionMode" >
    <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
   </property>
   <property name="maximum" >
    <number>4000</number>
   </property>
   <property name="minimum" >
    <number>600</number>
   </property>
   <property name="value" >
    <number>600</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelHeight" >
   <property name="geometry" >
    <rect>
     <x>160</x>
     <y>58</y>
     <width>46</width>
     <height>14</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>0</hsizetype>
     <vsizetype>0</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>Height:</string>
   </property>
   <property name="buddy" >
    <cstring>spinBoxHeight</cstring>
   </property>
  </widget>
  <widget class="QSlider" name="iterationSlider" >
   <property name="geometry" >
    <rect>
     <x>5</x>
     <y>16</y>
     <width>300</width>
     <height>32</height>
    </rect>
   </property>
   <property name="contextMenuPolicy" >
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="minimum" >
    <number>100000</number>
   </property>
   <property name="maximum" >
    <number>15000000</number>
   </property>
   <property name="singleStep" >
    <number>10000</number>
   </property>
   <property name="pageStep" >
    <number>100000</number>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition" >
    <enum>QSlider::TicksBelow</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>iterationSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelIterationsValue</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>85</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel" >
     <x>124</x>
     <y>10</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
